src/content/app/genome-browser/components/browser-location-indicator/BrowserLocationIndicator.test.tsx

```diff
@@ -73,6 +81,17 @@ describe('BrowserLocationIndicator', () => {
       expect(renderedName?.textContent).toBe(chrName);
     });
 
+    it('displays circular chromosome', () => {
+      const newstate = set(
+        'genome.genomeKaryotype.genomeKaryotypeData.human[0].is_circular',
+        true,
+        mockState
+      );
+      const { container } = renderComponent(newstate);
+      const circularIndicator = container.querySelector('.circularIndicator');
+      expect(circularIndicator).toBeTruthy();
+    });
```

Comment:
This test seems to be failing at the moment. From CI/CD logs:

![image](https://user-images.githubusercontent.com/6834224/150967248-826b4871-7ba4-4ed2-962b-edbb2b62e3fc.png)